Formula/tuntox.rb

```diff
@@ -0,0 +1,38 @@
+class Tuntox < Formula
+  desc "Tunnel TCP connections over the Tox protocol"
+  homepage "https://gdr.name/tuntox/"
+  url "https://github.com/gjedeer/tuntox/archive/refs/tags/0.0.10.1.tar.gz"
+  sha256 "7f04ccf7789467ff5308ababbf24d44c300ca54f4035137f35f8e6cb2d779b12"
+  license "GPL-3.0-only"
+  head "https://github.com/gjedeer/tuntox.git", branch: "master"
+
+  depends_on "cscope" => :build
+  depends_on "pkg-config" => :build
+  depends_on "toxcore"
+
+  def install
+    inreplace "gitversion.h", /.*/, '#define GITVERSION "N/A"'
+    inreplace "Makefile", "gitversion.h: FORCE", "gitversion.h:"
```

Comment:
Oops, I think we don't need `-lrt` on macOS.
```suggestion
    inreplace "Makefile" do |s|
      s.gsub! "gitversion.h: FORCE", ""
      s.gsub! "-lrt", "" if OS.mac?
    end
```